src/components/link.js

```diff
@@ -27,6 +27,7 @@ export default {
     replace: Boolean,
     activeClass: String,
     exactActiveClass: String,
+    ariaCurrentValue: 'page' | 'step' | 'location' | 'date' | 'time',
```

Comment:
Let's use the default in props and pass a correct type

```suggestion
    ariaCurrentValue: {
      type: String,
      default: 'page'
},
```

Reply:
Added the default, and it looks like the same hack you used for `toTypes` and `eventTypes` allows adding a literal type (which seems preferred to me, admittedly coming from the Typescript side of things)

Reply:
Unfortunately, it won't work 😅 
The codebase is used a quite outdated version of flow. For typing props in TS, there is a `PropsType` that would allow doing what you want. Here we do need to provide the `String` constructor for it to work with Vue

Reply:
Ah, that's unfortunate :P.

In any case, updated to `String` now!